tests/CppUTestExt/MockSupportTest.cpp

```diff
@@ -909,6 +939,388 @@ TEST(MockSupportTest, customObjectWithFunctionComparatorThatFailsCoversValueToSt
     CHECK_EXPECTED_MOCK_FAILURE_LOCATION(failure, __FILE__, __LINE__);
 }
 
+TEST(MockSupportTest, customTypeOutputParameterSucceeds)
+{
```

Comment:
No problem. I didn't put them there, I just didn't take the trouble to remove them.